src/applications/gqrx/receiver.cpp

```diff
@@ -178,14 +178,25 @@ void receiver::stop()
  */
 void receiver::set_input_device(const std::string device)
 {
-    qDebug() << "Set input device:";
-    qDebug() << "  old:" << input_devstr.c_str();
-    qDebug() << "  new:" << device.c_str();
-
     std::string error = "";
 
     if (device.empty())
         return;
+    if (input_devstr.compare(device) == 0)
+    {
+#ifndef QT_NO_DEBUG_OUTPUT
+        qDebug() << "No change in input device:";
+        qDebug() << "  old: " << input_devstr.c_str();
+        qDebug() << "  new: " << device.c_str();
+#endif
+        return;
+    }else{
```

Comment:
Looks like the rest of the code puts parentheses on their own lines. Should we update this to match?

Reply:
Fixed in 060e0af6750fc4c83f83041fd6b99fdef6420b2b.